From: Don Dutile <ddutile@redhat.com>
Date: Fri, 25 Jul 2008 15:15:32 -0400
Subject: [xen] ia64 PV: Makefile changes
Message-id: 488A2654.8090006@redhat.com
O-Subject: [RHEL5.3 PATCH 2/4]: Xen PV-on-HVM subsystem for bare metal/FV ia64 kernel
Bugzilla: 442991
RH-Acked-by: Bill Burns <bburns@redhat.com>
RH-Acked-by: Markus Armbruster <armbru@redhat.com>

BZ 442991 -- Include xenpv-driver in bare metal kernel rpm.

Simple change to the Makefile to use a relative path
to get the ia64-specific files, similar to the relative
paths used by other shared xen files.

See 0/4 for test details.

Please review & ACK.

- Don

diff --git a/drivers/xenpv_hvm/platform-pci/Makefile b/drivers/xenpv_hvm/platform-pci/Makefile
index 3aac3f6..eb18c1b 100644
--- a/drivers/xenpv_hvm/platform-pci/Makefile
+++ b/drivers/xenpv_hvm/platform-pci/Makefile
@@ -20,6 +20,6 @@ xen-platform-pci-objs += ../../xen/core/xen_proc.o
 
 # Can we do better ?
 ifeq ($(ARCH),ia64)
-  xen-platform-pci-objs += $(srctree)/arch/ia64/xen/xcom_mini.o 
-  xen-platform-pci-objs += $(srctree)/arch/ia64/xen/xencomm.o
+  xen-platform-pci-objs += ../../../arch/ia64/xen/xcom_mini.o 
+  xen-platform-pci-objs += ../../../arch/ia64/xen/xencomm.o
 endif
